ui-tests/src/test/java/org/openmrs/reference/MergePatientTest.java

```diff
@@ -29,27 +28,23 @@
 
     @Before
     public void setUp() throws Exception {
-
         testPatient = createTestPatient();
         testPatient1 = createTestPatient();
     }
-
+    
     @Test
     @Category(BuildTests.class)
     public void mergePatientByNameTest() throws Exception {
-
         DataManagementPage dataManagementPage = homePage.goToDataManagement();
         MergePatientsPage mergePatientsPage = dataManagementPage.goToMergePatient();
         mergePatientsPage.enterPatient1(testPatient.identifier);
         mergePatientsPage.enterPatient2(testPatient1.identifier);
         mergePatientsPage.clickOnContinue();
         mergePatientsPage.clickOnMergePatient();
         PatientVisitsDashboardPage dashboardPage = mergePatientsPage.clickOnContinue();
-
         assertThat(dashboardPage.getPatientFamilyName(), is(testPatient1.familyName));
     }
-
-
+    
```

Comment:
@HerbertYiga in response to your request to check this out, this does no technical revision

Reply:
> in response to your request to check this out, this does no technical revision

@k-joseph  well i find this working well,should we just get the ticket simply closed off,i really appreciate your efforts on the reviewing, its quick and keeps the cycle moving

Reply:
Its a good idea to close this pr since this test _was/is_ passing already. cc: @HerbertYiga  @k-joseph